docs/docker-compose.yml

```
@@ -1,11 +1,11 @@
 version: '3'
 services:
   npx:
-    image: node:16.18
+    image: node:20.11
     user: node
-    command: npx serve
+    command: npm start
```

Pas de `npm start` dans l'image `node`, mais tu peux faire comme suit : 
```suggestion
    command: bash -c "npm install -g live-server && npx live-server --port=4242"
```